system-addon/lib/TopSitesFeed.jsm

```diff
@@ -234,13 +234,9 @@ this.TopSitesFeed = class TopSitesFeed {
       case at.INIT:
         this.refresh();
         break;
-      case at.NEW_TAB_LOAD:
-        if (
-          // When a new tab is opened, if the last time we refreshed the data
-          // is greater than 15 minutes, refresh the data.
-          (Date.now() - this.lastUpdated >= UPDATE_TIME)
-        ) {
-          this.refresh(action.meta.fromTarget);
+      case at.SYSTEM_TICK:
+        if (Date.now() - this.lastUpdated >= UPDATE_TIME) {
+          this.refresh();
```

Comment:
I believe @k88hudson has mentioned that we want to avoid "random" updates to already opened pages. Looks like we probably want some way to refresh without dispatching to content now and only update chrome… maybe just `SendToMain`?

Reply:
@Mardak This goes for Highlights as well right?